readme/Readme.scala

```diff
@@ -65,6 +65,7 @@ object Readme {
 
   def versionBadge = {
     def timestampOfTag(tag: String): String = {
+      shell.call(s"git fetch origin refs/tags/$tag:refs/tags/$tag")
```

Comment:
or you could set up drone to do this from the beginning http://readme.drone.io/usage/cloning/